game.py

```diff
@@ -12,7 +12,17 @@ class Random(Bot):
     """
     Cory's bot :)
     """
-    pass
+
+    def move(self, state):
+        valid_pile_indexes = []
+        for i, pile in enumerate(state):
+            if pile >=1:
```

Comment:
Missing whitespace after operator

Reply:
Also @romainrichard, do you think all of the `1`s that are going to end up all over this file are worth defining a constant for? `MIN_VALID_MOVE = 1` They are going to be in pretty much every class for this same purpose.